ext/src/main/java/io/burt/kafka/clients/KafkaClientsLibrary.java

```diff
@@ -58,6 +59,8 @@ static RubyClass mapErrorClass(Ruby runtime, Throwable t) {
       String javaName = t.getClass().getSimpleName();
       String rubyName = javaName.substring(0, javaName.length() - 9) + "Error";
       return (RubyClass) runtime.getClassFromPath(String.format("Kafka::Clients::%s", rubyName));
+    } else if (t instanceof TimeoutException) {
+      return (RubyClass) runtime.getModule("Timeout").getConstant("Error");
```

Comment:
I was just looking into that, and I'm wondering if maybe we should use our own error class instead. It feels weird having to `require 'timeout'` just to be able to rescue an exception, doesn't it?